src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -334,7 +334,8 @@ void AttitudeEstimatorQ::task_main()
 
 		} else if (ret == 0) {
 			// Poll timeout, do nothing
-			PX4_WARN("Q POLL TIMEOUT");
+// TODO: RobD debugging - quiet this for now
```

Comment:
Not sure what the cause was during earlier development, but i've removed
(re-enabled the warning) it now and it doesn't seem to be occurring any
longer.

On 2 April 2016 at 00:13, Lorenz Meier <notifications@github.com> wrote:

> In src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp
> <https://github.com/PX4/Firmware/pull/4136#discussion_r58203116>:
>
> > @@ -334,7 +334,8 @@ void AttitudeEstimatorQ::task_main()
> >
> >  		} else if (ret == 0) {
> >  			// Poll timeout, do nothing
> > -			PX4_WARN("Q POLL TIMEOUT");
> > +// TODO: RobD debugging - quiet this for now
>
> This suggests a substantial issue with the driver - I can't accept this
> diff if this is really occurring.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/PX4/Firmware/pull/4136/files/aa24db919a11a5e10651b20ab15907b0f5185086#r58203116>
>